TypeCobol.LanguageServer.Test/ProcessorTests/DataLayout/CSV-copy.txt

```diff
@@ -0,0 +1,10 @@
+[{"line":1,"character":12}]
+---------------------------------------------------------------------------
```

Comment:
You can add some `-` to match the length of the longest line in file if you like. Same remark is valid for all test data files.

Reply:
Done